src/App.js

```diff
@@ -102,6 +160,7 @@ class App extends Component {
     return classes;
   };  // End propsToClasses()
 
+  /** Render the App component. */
```

Comment:
It may be worth asking if this comment is needed. i.e. does it clarify anything that the name doesn't already tell us?

Reply:
Yeah, probably not. I'll get rid of it.